src/components/graph-view.js

```diff
@@ -1235,7 +1235,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     const { edgeArrowSize, gridSpacing, gridDotSize, nodeTypes, nodeSubtypes, edgeTypes, renderDefs } = this.props;
     return (
       <div
-        className="view-wrapper"
+        className="react-digraph"
```

Comment:
Please avoid changing the existing class names. You may add `react-digraph`, but don't remove `view-wrapper`, otherwise that would be a breaking change and would require another major version bump.